pyomo/contrib/pynumero/interfaces/pyomo_grey_box_nlp.py

```diff
@@ -364,11 +388,22 @@ def get_constraints_scaling(self):
     # overloaded from NLP
     def evaluate_objective(self):
         # objective is owned by the pyomo model
-        return self._pyomo_nlp.evaluate_objective()
+        obj = 0
+        for nlp in self._nlps:
+            obj += nlp.evaluate_objective()
+        return obj
```

Comment:
Can you add a comment noting that we've already checked to make sure there's only one objective?